src/components/OrderDetails/OrderDetails.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+
+import {ThemeProvider} from '../ThemeProvider';
+import {Card} from '../Card';
+import {TextContainer} from '../TextContainer';
+import {Thumbnail} from '../Thumbnail';
+import {Stack} from '../Stack';
+import {Button} from '../Button';
+import {TextStyle} from '../TextStyle';
+
+export interface OrderDetailsProps {}
+
+export class OrderDetails extends React.PureComponent<OrderDetailsProps> {
+  render() {
+    return (
+      <ThemeProvider
+        theme={{
+          colorScheme: 'dark',
+          colors: {
+            surface: '#020c1d',
+            secondary: '#103560',
+            interactive: '#009973',
+          },
+        }}
+      >
+        <Card title="Order details">
+          <Card.Section title="Unfulfilled">
+            <Stack alignment="center" distribution="equalSpacing">
+              <Stack alignment="center">
+                <Thumbnail
```

Comment:
I suspect the example you posted wasn't using a thumbnail. I think this is a good approach though. :)